notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Some other exercises might be to choose another Hubble spectrum - maybe one of the ones with lower exposure time, but the same instrument/filter, and download and plot it, and compare to the tutorial one. (Presumably the lower exposure time one will have lower S/N. If it is from a significantly different epoch, could it also show notable differences in the strength of any of the emission features? Presumably not, but you could ask the question!) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>